src/soft404s/constants.js

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright 2025 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+export const SOFT_404_INDICATORS = [
+  'page not found',
+  'this page does not exist',
+  'not found',
+  'no longer available',
+  "we couldn't find",
+  'nothing here',
+  'page got',
+  'page not',
+  'whoops',
+  'invalid page',
+  'invalid',
+  'page you are looking for',
+  '404',
```

Comment:
`404` is generic. A page can be < 500words and have "404 reviews" phrase present. In that case a false positive can be triggered.